ArduPlane/ArduPlane.cpp

```diff
@@ -49,8 +49,8 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(update_alt,             10,    200),
     SCHED_TASK(adjust_altitude_target, 10,    200),
     SCHED_TASK(afs_fs_check,           10,    100),
-    SCHED_TASK(gcs_update,             50,    500),
-    SCHED_TASK(gcs_data_stream_send,   50,    500),
+    SCHED_TASK_CLASS(GCS,                 (GCS*)&plane._gcs,       update,                 50,  500),
+    SCHED_TASK_CLASS(GCS,                 (GCS*)&plane._gcs,       data_stream_send,       50, 1000),
```

Comment:
Execution time estimate changes here for this, and is surprisingly different between the different vehicles. Is that a real difference?

Reply:
Fixed.

Reply:
.... to be clear, I fixed the incorrect change here.

The delta between the vehicles... well, their data streams are different, I guess...

I wouldn't worry about this too much - when sending is moved to a thread this stuff will go away.